backend/src/main/resources/application.yaml

```diff
@@ -22,6 +22,9 @@ spring:
   session:
     store-type: jdbc
     timeout: 15m
+    jdbc:
+        initialize-schema: never
+        table-name: ${spring.jpa.properties.hibernate.default_schema}.spring_session
```

Comment:
That's a good question, but I don't think so. (The reason I think not is that liquibase didn't throw up a million errors that something else was modifying the database, which it did the one time I set this to `initialize-schema: always` while testing 😄 ) @benwarfield-usds has better context on this, but I believe that the previous deployment just set us up to have in-memory spring sessions, rather than ones backed by our database.